beetsplug/vocadb.py

```diff
@@ -741,10 +819,12 @@ def get_album_track_infos(
     def get_artists(
         cls,
         artists: list[AlbumArtistDict],
-        va_string: str = DEFAULT_VA_STRING,
+        va_string: str = default_config["va_string"],
```

Comment:
It feels a bit odd to me to read this from the `default_config`.

Since every time we call `get_artists`, we specify the `va_string` this could just be made a required argument:
```suggestion
        va_string: str,
```

Or, this whole argument could just be dropped, and we can get `va_string` by doing
`va_string: str = cls.va_string`